### ISSUE TITLE
Matek F722-SE no camera image

### ISSUE THREAD

**Issue by KiloTango213** [iNavFlight/inav#4800]
Hello there,

I am using a Matek F722-SE FC with iNav and for now everything worked great.
But i have a problem with the VTX und C1 conections.

I connected my vtx transmitter to VBat, Gnd and VTX and my runcam micro eagle to c1 5v and gnd, just like the describtion in the matek manual. 

Problem is I dont get an image to my receiver. 
However if I connect just the video channel directly from camera to vtx transmitter  I get an image.

I played with User 1 and User2 to activate the transmitter and camera wich worked fine, but still no image.

So my question now is: Is there some sort of switch to enable c1 or anything else I missed and could do?

**Comment by DzikuVx** [comment]
Check if connecting RX and having radio turned on helps.

**Comment by KiloTango213** [comment]
Thanks for the tip. I checked it twice but it didnt fix it.
I talked to Matek Support and within 1 hour and sending some pictures, they told me to check back with the dealer to get a new one. They replacement is on its way to me.

I still think that the problem is with the osd output, maybe the video will not come through without osd. So I will test that next with the runcam osd adapter. It's on its way aswell.
